README.md

```diff
@@ -223,5 +223,18 @@ A collection of experiments and research to learn about what makes an Alaveteli
         <strong>Predominantly social media.</strong> Sites with a strong journalistic focus get a greater proportion of traffic from news and campaign sites.
       </td>
     </tr>
+    <tr>
+      <td>
+        <a href="experiments/effect-of-listing-successes-on-asktheeu-homepage.md">
+          We currently show a curated list of requests on the homepage, would
+          visitors be inspired to make their own requests if we showed them
+          recent successes instead?
+        </a>
+      </td>
+      <td>
+        <strong>Probably not.</strong> There was no noticeable difference in the
+        rate of new requests between the 2 variations of the homepage.
```

Comment:
s/Probably not/No/ – We didn't see an increase. Also doesn't need the explainer IMO;